src/target/adiv5.c

```diff
@@ -476,7 +476,7 @@ static uint32_t cortexm_initial_halt(adiv5_access_port_s *ap)
  */
 static bool cortexm_prepare(adiv5_access_port_s *ap)
 {
-#if PC_HOSTED == 1 || ENABLE_DEBUG == 1
+#if (defined(PC_HOSTED) && PC_HOSTED == 1) || (ENABLE_DEBUG == 1)
```

Comment:
PC_HOSTED should always be defined - the main Makefile should be ensuring it's 0 for non-BMDA platforms. Same comment about parens.